db_stress_tool/db_stress_driver.cc

```diff
@@ -81,31 +83,6 @@ bool RunStressTestImpl(SharedState* shared) {
   stress->InitDb(shared);
   stress->FinishInitDb(shared);
 
-  if (FLAGS_write_fault_one_in) {
-    if (!FLAGS_sync_fault_injection) {
-      // unsynced WAL loss is not supported without sync_fault_injection
-      fault_fs_guard->SetDirectWritableTypes({kWalFile});
-    }
-    IOStatus error_msg;
-    if (FLAGS_inject_error_severity <= 1 || FLAGS_inject_error_severity > 2) {
-      error_msg = IOStatus::IOError("Retryable injected write error");
-      error_msg.SetRetryable(true);
-    } else if (FLAGS_inject_error_severity == 2) {
-      error_msg = IOStatus::IOError("Fatal injected write error");
-      error_msg.SetDataLoss(true);
-    }
-    // TODO: inject write error for other file types including
-    //  MANIFEST, CURRENT, and WAL files.
-    fault_fs_guard->SetRandomWriteError(
-        shared->GetSeed(), FLAGS_write_fault_one_in, error_msg,
-        /*inject_for_all_file_types=*/false, {FileType::kTableFile});
-    fault_fs_guard->SetFilesystemDirectWritable(false);
-    fault_fs_guard->EnableWriteErrorInjection();
-  }
```

Comment:
Write injection is now done thread-local way. So move into each OperateDb thread - see below db_stress_test_base.cc.